pypsa-to-imac

```diff
@@ -0,0 +1,400 @@
+                 n.links.p_nom_opt.filter(like ='gas CHP CC').filter(like =country)).sum()            
+            
+            #Capacity :Hydrogen
+            
+            var['Installed capacity|Heat|Gases|Hydrogen'] = 0.001* (n.links.efficiency2.filter(like ='H2 Fuel Cell').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='H2 Fuel Cell').filter(like =country)).sum()  
+            var['Installed capacity|Heat|Gases|Hydrogen'] += 0.001* (n.links.efficiency3.filter(like ='Fischer-Tropsch').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='Fischer-Tropsch').filter(like =country)).sum()    
+            
+            #Capacity :Power to gas (?)
+            """    
+            var['Installed capacity|Heat|Gases|P2G'] = 0.001* (n.links.efficiency.filter(like ='Sabatier').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='Sabatier').filter(like =country)).sum()*var['Installed capacity|Heat|Gases|Fossil|Natural Gas']/\
+                 (var['Installed capacity|Heat|Gases|Fossil|Natural gas']+var['Installed capacity|Electricity|Gases|Fossil|Natural gas']                
+            """
+            #Capacity : Oil
+ 
+            var['Installed capacity|Heat|Liquids|Fossil'] = 0.001*(n.links.efficiency.filter(like ='oil boiler').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='oil boiler').filter(like =country)).sum() 
+            
+            #Capacity : solar thermal
+    
+            var['Installed capacity|Heat|Solar'] = 0.001*(n.generators.p_nom_opt.filter(like ='solar thermal').filter(like =country)).sum()
+            
+                                                
+            # Hydrogen capcaties 
+
+            var['Installed capacity|Hydrogen|Electricity'] = 0.001* (n.links.efficiency.filter(like ='H2 Electrolysis').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='H2 Electrolysis').filter(like =country)).sum()    
+                  
+            var['Installed capacity|Hydrogen|Gasses|Fossil|Natural gas|CCS'] = 0.001* (n.links.efficiency.filter(like ='SMR CC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='SMR CC').filter(like =country)).sum()                                                                       
+
+            var['Installed capacity|Hydrogen|Gasses|Fossil|Natural gas'] = 0.001* (n.links.efficiency.filter(like ='SMR').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='SMR').filter(like =country)).sum()               
+                  
+            var['Installed capacity|Hydrogen'] = var['Installed capacity|Hydrogen|Electricity']+\
+            var['Installed capacity|Hydrogen|Gasses|Fossil|Natural gas']
+                  
+            # P2G capcaties 
+                  
+            var['Installed capacity|P2G|Electricity'] = 0.001* (n.links.efficiency.filter(like ='helmeth').filter(like =country)*\
```

Comment:
Acc. to IAM team, it means "the power capacity used to produce H2, synthetic methane, synthetic oil or the three of them". So it can include Sabatier , Helmeth (power to methane), Electrolysis (water to hydrogen), and Fischer-Tropsch.